From 34f9205645210cb2bb0939cc2d725f9f2b1079cf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Tue, 2 Nov 2021 13:28:24 +0100
Subject: [PATCH 314/456] arm64: dts: rk3399-pinephone-pro: Add chassis-type =
 "handset" to DT

This way, desktop environment can pick up device type from DT.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index ad940bb07ba0..ed2f3ba75369 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -17,6 +17,7 @@
 / {
 	model = "Pine64 PinePhonePro";
 	compatible = "pine64,pinephone-pro", "rockchip,rk3399";
+	chassis-type = "handset";
 
 	chosen {
 		bootargs = "earlycon=uart8250,mmio32,0xff1a0000";
-- 
2.34.1

